libs/initScripts/local.js

```diff
@@ -137,6 +142,48 @@ const run = async () => {
       case 'update-userreplicasetmanager-init-config':
         await _updateUserReplicaSetAddresses(ethAccounts)
         break
+
+      case 'update-user-replica-set':
+        console.log(`Usage: node local.js update-user-replica-set userId=1 primary=2 secondaries=3,1`)
```

Comment:
nit - maybe make this a comment instead of a log

Reply:
i actually think its better as a log - that way it spits out the usage every time